common/crypto/attestation-api/scripts/enclave_to_mrenclave.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Yes, Marcus also suggested that -- as a separate PR.

Reply:
didn't see that in the discussion above.  Given that you introduced the duplication here, it would seem to me more natural (and efficient) to just fix it here but if you prefer to do it in a separate PR, that should work as well ...

Reply:
It was in the meeting.